polkadot-parachains/encointer-runtime/src/lib.rs

```diff
@@ -152,6 +154,81 @@ const NORMAL_DISPATCH_RATIO: Perbill = Perbill::from_percent(75);
 /// We allow for 500 ms of compute with a 6 second average block time.
 const MAXIMUM_BLOCK_WEIGHT: Weight = WEIGHT_PER_SECOND / 2;
 
+//TODO add meaningful values
+parameter_types! {
+	pub const ProxyDepositBase: Balance = 32;
+	pub const ProxyDepositFactor: Balance = 32;
+	pub const MaxProxies: u16 = 32;
+	pub const AnnouncementDepositBase: Balance = 32;
+	pub const AnnouncementDepositFactor: Balance = 32;
+	pub const MaxPending: u16 = 32;
+}
+
+/// The type used to represent the kinds of proxying allowed.
+#[derive(
+	Copy,
+	Clone,
+	Eq,
+	PartialEq,
+	Ord,
+	PartialOrd,
+	Encode,
+	Decode,
+	RuntimeDebug,
+	TypeInfo,
+	MaxEncodedLen,
+)]
+pub enum ProxyType {
+	Any,
+	NonTransfer,
+	BazaarEdit,
```

Comment:
I think we might wan to have a proxy for `BazaarRoot`, which could also update and delete a business. What's your take on this @brenzi ?

Reply:
ww actually need this because an anon proxy has no private key, wo all actions involving a business must be proxyable. but please don't use root, rather BazaarOwner

Reply:
How would the BazaarOwner be different to a NonTransfer? Or should it only be capabe of precisely manipulate Bazaar related stuff and no other thing at all?

Reply:
exactly. but, now that you mention it, businesses will never need to hold a lot of tokens and governance will be per capita, not per capital, so non-transfer is actually fine

Reply:
wouldn't it make sense to exclude proxies from governance at all, then?

Reply:
later, maybe. don't bother for now